connectors/slack/template/index.js

```diff
@@ -134,21 +137,10 @@ async function dispatch(ctx) {
 }
 
 async function registerCore(ctx, isDeregister) {
```

Comment:
Split this into registerCore and releaseCore to avoid the awkward boolean modality?

Reply:
If you have to keep the modality, invert it so that it's a positive test ("isRegistering") rather than the awkward "!isDeregister".  Easier on the brain.

Reply:
Wont't fix, don't have time to refactor this now